fury/geometry.py

```diff
@@ -228,3 +232,35 @@ def create_text(text, material, **kwargs):
 
     text = Text(text=text, material=material, **kwargs)
     return text
+
+
+def create_image(image_input, material, **kwargs):
```

Comment:
Add a check to ensure it’s a 2D grayscale or normalized float32 image. and auto-normalize or raise erorr if outside [0 ,1] .